earth_enterprise/SConstruct

```diff
@@ -169,6 +169,12 @@ stage_bin = env.PhonyTargets(**stage_bin_cmds)
 stage_data = env.Alias('stage_data', installdirs['root'])
```

Comment:
Change Copyright at top of file to add `Copyright 2019 Open GEE Contributors`

Reply:
done